vng_api_common/client.py

```diff
@@ -25,20 +75,38 @@ def get_client(url: str, url_is_api_root=False) -> Optional[Client]:
         client_getter = import_string(custom_client_fetcher)
         return client_getter(url)
 
-    # default implementation
-    Client = import_string(settings.ZDS_CLIENT_CLASS)
+    from zgw_consumers.client import build_client
+    from zgw_consumers.models import Service
 
-    if url_is_api_root and not url.endswith("/"):
-        url = f"{url}/"
+    # default implementation
+    client_class = import_string(settings.CLIENT_CLASS)
+    service: Optional[Service] = Service.get_service(url)
 
-    client = Client.from_url(url)
-    if client is None:
+    if not service:
+        logger.warning(f"No service configured for {url}")
         return None
 
-    APICredential = apps.get_model("vng_api_common", "APICredential")
+    return build_client(service.api_root, client_factory=client_class)
+
+
+def get_auth_headers(
```

Comment:
I think we should be able to make use of https://github.com/maykinmedia/zgw-consumers/blob/main/zgw_consumers/client.py#L91